src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/OrderStates/OrderStateType.php

```diff
@@ -0,0 +1,207 @@
+                    'constraints' => [
+                        new TypedRegex([
+                            'type' => 'generic_name',
+                        ]),
+                    ],
+                ],
+            ])
+            ->add('color', ColorPickerType::class, [
+                'required' => false,
+            ])
+            ->add('logable', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Consider the associated order as validated.', 'Admin.Shopparameters.Feature'),
+                'attr' => [
+                    'material_design' => true,
+                ],
+            ])
+            ->add('invoice', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Allow a customer to download and view PDF versions of his/her invoices.', 'Admin.Shopparameters.Feature'),
+                'attr' => [
+                    'material_design' => true,
+                ],
+            ])
+            ->add('hidden', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Hide this status in all customer orders.', 'Admin.Shopparameters.Feature'),
+                'attr' => [
+                    'material_design' => true,
+                ],
+            ])
+            ->add('send_email', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Send an email to the customer when his/her order status has changed.', 'Admin.Shopparameters.Feature'),
+                'attr' => [
+                    'material_design' => true,
+                ],
+            ])
+            ->add('pdf_invoice', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Attach invoice PDF to email.', 'Admin.Shopparameters.Feature'),
+                'attr' => [
+                    'material_design' => true,
+                ],
+            ])
+            ->add('pdf_delivery', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Attach delivery slip PDF to email.', 'Admin.Shopparameters.Feature'),
+                'attr' => [
+                    'material_design' => true,
+                ],
+            ])
+            ->add('shipped', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Set the order as shipped.', 'Admin.Shopparameters.Feature'),
+                'attr' => [
+                    'material_design' => true,
+                ],
+            ])
+            ->add('paid', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Set the order as paid.', 'Admin.Shopparameters.Feature'),
+                'attr' => [
+                    'material_design' => true,
+                ],
+            ])
+            ->add('delivery', CheckboxType::class, [
```

Comment:
Does this naming need to be changed?

Reply:
it's the same name as the DB column. It doesnt affect the display so I would keep it

WDYT @PrestaShop/prestashop-core-developers ?

Reply:
Better keep it then as it will help developers who discover the migrated page to map things between Symfony and legacy